src/components/Filter/Filter.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect } from 'react';
+
+function Filter(props) {
+  const [status, setStatus] = useState(false); // States to open additional filters
```

Comment:
create hooks folder and use useReducer

Reply:
Sure

Reply:
Could you check please if there is more space to use Custom Hooks comparing what I chose with "use filter"?